README.md

```diff
@@ -45,6 +45,13 @@ xComputer resource has following properties:
 
 ## Versions
 
+
+### 1.3.0
+
+* xComputer
+    * Fix issue with Test-TargetResource when not specifying Domain or Workgroup name
+    * Added tests
```

Comment:
I would not include it to release notes, since it's just a development artifact. On the other hand, we ship tests together with code and they can used as an examples. So it's up to you. :)